utils/rsvp.js

```diff
@@ -0,0 +1,27 @@
+import {
+  Player,
+  Game,
+  Location,
+  GamePlayer,
+  Rsvp,
+  SkillLevel,
+} from "../src/models";
+
+async function rsvp(gameId, playerId, newRsvp) {
+  const original = await DataStore.query(GamePlayer, (c) => c.and(c => [
```

Comment:
try catch

Reply:
Surround this whole function in a try catch if you must, but every API call needs a try catch. Otherwise if a call doesn't go through, our entire app crashes or an error message pops up at the bottom of the screen in Expo, which we don't want.